pyrightconfig.json

```diff
@@ -1,14 +1,14 @@
 {
     "typeCheckingMode": "basic",
     "include": [
-        "astropy/units"
+        "astropy/units/format"
     ],
     "exclude": [
-        "astropy/units/tests"
+        "astropy/units/format/vounit.py"
     ],
     "executionEnvironments": [
         {
-            "root": "astropy/units",
```

Comment:
the idea here was to explicitly create subpackage-level settings. I'm not sure why you changed it @mhvk, can you explain ?

Reply:
I thought I just got errors if I left it in-place, but will admit I was slightly randomly changing things... 

Anyway, I just checked and that is actually not a problem, so fine to revert that change! The one question then is how to do parts of a package, since the `include` and `exclude` seem to be `astropy` wide rather than per execution environment...